hyperspy/_components/logistic.py

```diff
@@ -17,87 +17,58 @@
 # along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
 
 
-import numpy as np
+from hyperspy._components.expression import Expression
 
-from hyperspy.component import Component
 
 
-class Logistic(Component):
+class Logistic(Expression):
 
-    """Logistic function component
+    """Logistic function (sigmoid/s-shaped curve) component
 
-    f(x) = a/(1+b*exp(-c*(x-origin)))
+    .. math::
+    
```

Comment:
Use latex to display nicely the formula (see for example https://github.com/hyperspy/hyperspy/blob/RELEASE_next_minor/hyperspy/_components/lorentzian.py#L30)?

Reply:
Good idea. I also improved the docstring further for consistency.